device/alpaka/include/traccc/alpaka/seeding/track_params_estimation.hpp

```diff
@@ -58,7 +59,7 @@ struct track_params_estimation
     /// Memory resource used by the algorithm
     traccc::memory_resource m_mr;
     /// Copy object used by the algorithm
-    vecmem::copy& m_copy;
+    vecmem::device_copy& m_copy;
```

Comment:
Why would you need to use a specific type here all of a sudden? 😕 We use the generic `vecmem::copy` type in the CUDA and SYCL algorithms as well.

The important part is to **instantiate** the correct type. Not to have reference to the correct type.